src/components/navbar.js

```diff
@@ -0,0 +1,35 @@
+import Image from "next/image";
```

Comment:
Same here. Make it take the names of the different items as inputs

Reply:
This change has been done